x/auth/params.go

```diff
@@ -15,18 +14,18 @@ const DefaultParamspace = "auth"
 
 // Default parameter values
 const (
-	DefaultMemoCostPerByte        sdk.Gas = 3
-	DefaultMaxMemoCharacters      uint64  = 256
-	DefaultTxSigLimit             uint64  = 7
-	DefaultSigVerifyCostED25519   uint64  = 590
-	DefaultSigVerifyCostSecp256k1 uint64  = 1000
+	DefaultMaxMemoCharacters      uint64 = 256
+	DefaultTxSigLimit             uint64 = 7
+	DefaultTxSizeCostPerByte      uint64 = 10
```

Comment:
This is more or less an arbitrary value -- perhaps warrants more discussion on what the default value should be.

Reply:
We probably need to pick a minimum recommended gas price and guess some token price to really choose this

Reply:
- Given an average tx size (e.g. single I/O send with an average amount) of 250 bytes
- Given a token price of $1
- Given a recommended minimum gas price of 0.000004
- Given gas costs w/o accounting for tx size (including memo) for such a tx is 26300

we find that a cost of 10 per tx byte yields a fee of ~11c. We can increase the cost if we expect minimum gas prices to be lower.